src/Microsoft.OData.Core/UriParser/Resolver/NormalizedModelElementsCache.cs

```diff
@@ -89,26 +93,76 @@ public List<IEdmTerm> FindTerms(string qualifiedName)
             return null;
         }
 
+        /// <summary>
+        /// Find all navigation sources that match the <paramref name="name"/>.
+        /// </summary>
+        /// <param name="name">The case-insensitive name to match.</param>
+        /// <returns>A list of matching navigation sources, or null if no navigation source matches the name.</returns>
+        public List<IEdmNavigationSource> FindNavigationSources(string name)

```

Comment:
This method returns `List<IEdmNavigationSource>` and also outs `List<IEdmNavigationSource>`. Can we use one or the other?

Reply:
This method returns `List<IEdmNavigationSource>`, it does not "out" it, the signature is:
```c#
List<IEdmNavigationSource FindNavigationSources(string name);
```
and not
```c#
List<IEdmNavigationSource FindNavigationSources(string name, out List<IEdmNavigationSource> ...)
```

Reply:
Yea. I thought I saw the out param in the method signature.